.github/workflows/update_submodule.yml

```diff
@@ -0,0 +1,40 @@
+name: Get latest version of submodules and push back to 6-dev branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout
+      - uses: actions/checkout@v3
+        with: 
+          ref: 6-dev
+      
+      # Pull latest plone.api
+      - name: Pull latest plone.api
+        run: |
+          cd submodules/plone.api
+          git pull
+
+      # Pull latest plone.restapi
+      - name: Pull latest plone.restapi
+        run: |
+          cd submodules/plone.restapi
+          git pull
+
+      # Pull latest Volto
+      - name: Pull latest Volto
+        run: |
+          cd submodules/volto
```

Comment:
I am not sure whether `cd` resets to the root of `documentation` for each command. I guess we will see.

Reply:
It does.